src/components/Header/PageSettings/AccountNavigation/AccountNavigation.js

```diff
@@ -9,8 +9,8 @@ const AccountNavigation = (props) =>
     <Link to={'/404'}>
       <span>my account</span>
     </Link>
-    <Link to={'/404'}>
-      <span>wish list</span>
+    <Link to={'/wishlist'}>
```

Comment:
I belive brackets are unnecessary when you pass string prop

Reply:
And double quotes must be used here